scopus-import/src/test/java/no/sikt/nva/scopus/conversion/ScopusDocumentTest.java

```diff
@@ -16,4 +19,14 @@ void shouldParseScopusXmlToDocTp() {
 
         assertDoesNotThrow(() -> JAXB.unmarshal(new StringReader(file), DocTp.class));
     }
+
+    @Test
+    void some() {
```

Comment:
?

Reply:
Sorry, forgetting to delete this one every time i am testing :)